Examples/DotNetCoreBuild/test/TestDirectories/testDirectories.dsc

```diff
@@ -68,6 +68,25 @@ export const writeHardLinkInSharedOpaqueDirectoryIsAllowed = !Context.isWindowsO
     linkFileIntoDirectory(f`module.config.dsc`, sod, true);
 })();
 
+@@public
+export const moveDirectoryInsideSOD = !Context.isWindowsOS() && (() => {

```

Comment:
Please use context.getcurrenthost this property should be deprecated since it doesn't scale to other os'

Reply:
will change throughout...